wkcuber/utils.py

```diff
@@ -33,21 +33,16 @@
 logger = getLogger(__name__)
 
 
-def open_wkw(info, **kwargs):
-    if hasattr(info, "dtype"):
-        header = wkw.Header(np.dtype(info.dtype), **kwargs)
-    else:
-        header = wkw.Header(np.uint8, **kwargs)
```

Comment:
Is it correct that this `if` is no longer needed, because the header always has a `voxel_type` attribute?

Reply:
Yes, this should be correct!